hbp_knowledge/huntington/loi2018.bel

```diff
@@ -0,0 +1,203 @@
+#######################
```

Comment:
the curation from loi2018 needs to be deleted from this file 
here only dechristo2019 is curated
loi2019 is already merge into the master

Reply:
@sgebel  yes I tried doing that first, but upon sending the BEL file for Travis check, it showed me a build error stating that loi2018 is missing and hence I had to add it again.